### ISSUE TITLE
Preview + Export Not rendering

### ISSUE THREAD

**Issue by tay-lew** [bodymovin/bodymovin-extension#102]
Hi @bodymovin!

Although the animation is represented in AE, the preview and json file are populating through the extension. Please let me know if you have a solve! 

Was unable to attach the zip so have the folder uploaded to this wetransfer link: https://we.tl/t-R3iTMghAA0

Thanks in advance!

**Comment by bodymovin** [comment]
hi, can you share more info? Not sure I understand the problem

**Comment by tay-lew** [comment]
yes, the animation is preview-able in after effects, but when I got to preview and export through bodymovin, it won't show up, it's just blank, both in the preview window and the json file. 
<img width="966" alt="Screen Shot 2021-11-23 at 2 53 42 PM" src="https://user-images.githubusercontent.com/94928562/143094422-0d7be4c3-2858-4632-a45c-caa98bcadec4.png">
!
<img width="651" alt="Screen Shot 2021-11-23 at 2 54 33 PM" src="https://user-images.githubusercontent.com/94928562/143094673-53bbbd24-7d2c-49ed-9a41-4183cab41aa3.png">

**Comment by bodymovin** [comment]
video layers are not supported unfortunately

**Comment by tay-lew** [comment]
thank you @bodymovin ! okay, i replaced the video with a sequence of .png images but am still having the same problem. Attaching a link to the updated after effects file, please let me know if you're able to identify the root of the issue! https://we.tl/t-2Yr7v7zOr5 and thanks again!
